packages/excavator-script/src/projects/monsterParts.ts

```diff
@@ -224,13 +261,27 @@ function spadeMonsterParts(
     // eslint-disable-next-line libram/verify-constants
     haveEquipped($item`Everfull Dart Holster`)
   ) {
+    const dartParts = Object.keys(dartPartsToSkills());
     data.push(
-      ...Object.keys(dartPartsToSkills()).map((part) => ({
-        monster,
-        part,
-        confirmation: true,
-        source: "Everfull Dart Holster",
-      })),
+      ...monsterParts
+        .filter((part) => !dartParts.includes(part))
+        .map((part) => ({
+          monster,
+          part,
+          confirmation: false,
+          source: "Everfull Dart Holster",
+        })),
+    );
+
+    data.push(
```

Comment:
this was a bug in the original too, but if you have the perk for detecting butts, excavator shouldn't send data for butts